src/scss/themes/_base.scss

```diff
@@ -81,8 +81,8 @@
   --deemphasized-text-color: #{$deemphasized-color};
   --focus-outline: #{$focus-outline};
 
-  --very-deemphasized-link-color: #{rgba($anchor-color, 0.6)};
-  --very-deemphasized-text-color: #{rgba(#666, 0.6)};
+  --very-deemphasized-text-color: #757575;
+  --very-deemphasized-link-color: var(--very-deemphasized-link-color);
```

Comment:
Typically I try not to mix CSS vars and SASS vars in the `scss` files but I guess it doesn't really matter.

Reply:
Actually this is a circular reference. The CSS var is referencing itself. It looks like the browser treats this as a no-op.

Apparently this causes links inside of statuses in notifications (e.g. "so-and-so favorited your toot") to appear the same color as the rest of the text (e.g. `example.com` below):

![Screenshot from 2022-12-10 11-06-08](https://user-images.githubusercontent.com/283842/206871388-5760c450-cf4f-40a8-8541-c2d9e89dfa3c.png)